airbyte-webapp/src/views/Connector/ConnectorDocumentationLayout/ResourceNotAvailable/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { faArrowDown } from "@fortawesome/free-solid-svg-icons";
```

Comment:
We tend not to put actual code in the `index` files but just reexport it (since it's harder to navigate to those files if the actual code is inside the index). Could this please be either in `ResourceNotAvailable/ResourceNotAvailable.tsx` (and just reexported from `ResourceNotAvailable/index.ts`) or just in a `ResourceNotAvailable.tsx` inside `ConnectorDocumentationLayout`.

Reply:
Makes sense! Will follow this approach from now on